### DISCUSSION TITLE
How to group routes with locales (ex. /en or /es...)?

### DISCUSSION THREAD

**Discussion by alexander-bet** [apiato/apiato#748] [Q&A]
Hi,
I like the Apiato but I'm just starting to learn it. So, I have a question about the localization routes. Without Apiato I did smth like that:

`Route::prefix('{locale?}')
    ->middleware('localized')
    ->group(function() {
        Route::get('/', function () {
            return view('welcome');
        });
    });`

But now with Apiato we should place each route in separate files in different containers... And is there way to group routes with locales without writing middleware and prefix in each sepatate file?

**Comment by Mohammad-Alavi** [comment]
Hello, and welcome!

I haven't done this but I think this might help.
https://laravel.com/docs/urls#default-values
